src/benchparse.py

```diff
@@ -68,8 +68,8 @@ def write_rule(self, rule_id, stig_id, rule_sev, rule_title,
         outf.write('      command: "true"\n')
         if check_type.upper() == 'AUDIT':
             outf.write('      register: result\n')
-            outf.write('      changed_when: no\n')
             outf.write('      check_mode: no\n')
+        outf.write('      changed_when: no\n')
```

Comment:
I suppose we could keep this the old way if we always want to show "not implemented" patches as "changed," though I'd then also want to add a variable to skip these "not implemented" tasks by default.